### DISCUSSION TITLE
❓ Question around loading Sections and Containers

### DISCUSSION THREAD

**Discussion by perks98** [elementor/elementor#20215] [Developers Community]
My question is if the Flexbox Container experiment is enabled on a website already completely built with sections, inner sections and columns, when adding a new area in which it will use the containers will it load more bloat of CSS and JavaScript as it accommodates for both on the page?

**Comment by nicholaszein** [comment]
Hello @perks98!

The short answer is no. The new Container was built exactly to accommodate backward compatibility. Nothing will change in terms of assets. Of course, if you convert Sections to Containers, you would benefit greatly from the optimized DOM size, since Containers load fewer DOM elements and have more streamlined CSS, but it's up to you, to choose when to convert your sections.
